pywatts/core/base_step.py

```diff
@@ -89,13 +90,13 @@ def get_result(self, start: pd.Timestamp, end: Optional[pd.Timestamp], buffer_el
 
         # Check if the cached results fits to the request, if yes return it.
         if self.cached_result["cached"] is not None and self.cached_result["start"] == start and self.cached_result[
-            "end"] == end:
+            "end"] == end and self.cached_result["minimum_data"] == minimum_data:
```

Comment:
`self.cached_result["minimum_data"]` gets initialized but never reassigned, i.e. it is always `None`